src/server/rpc/procedures/geolocation/geolocation.js

```diff
@@ -58,7 +58,12 @@ if(!process.env.GOOGLE_GEOCODING_API) {
     };
 
 
-    // geocode an address and send back the details
+    /**
+     * Geolocates the address and sends back the coordinates
+     * @param {string} address target address
+     * @returns {structuredData}
```

Comment:
I would just say the javascript form for these things here (ie, `Object`)

Reply:
The idea is that the RPCs should be able to be written just as regular javascript with minimal additional knowledge required about the rest of the system and then netsblox will just automatically convert between JavaScript and NetsBlox types as needed.